content/_partials/changes.html.haml

```diff
@@ -0,0 +1,33 @@
+- page.changes.each do | change |
+  %li{:class => change.type }
+    = change.message
+    - if change.issue
+      (
+      %a{:href => "https://issues.jenkins-ci.org/browse/JENKINS-#{change.issue}" }<>
+        = "issue #{change.issue}"
+      )
+    - elsif change.pull
+      (
+      %a{:href => "https://github.com/jenkinsci/jenkins/pull/#{change.pull}" }<>
+        = "pull #{change.pull}"
+      )
+    - if change.references
+      - change.references.each_with_index do | reference, index |
+        - if index == 0
+          (
+        - else
+          ,&nbsp;
+        - if reference.issue
+          %a{:href => "https://issues.jenkins-ci.org/browse/JENKINS-#{reference.issue}" }<>
```

Comment:
The logic here should ideally check a custom prefix (e.g. "SECURITY"), but not required right now